web/src/app/projects/repository/repository.component.html

```diff
@@ -10,30 +10,144 @@
     </span>
   </mat-card-header>
   <mat-divider></mat-divider>
-  <mat-card-content fxLayout="row">
-    <mat-list fxFlex dense>
-      <h3 mat-subheader>Events</h3>
-      <mat-list-item *ngFor="let event of repository.events | slice:0:5">
-        <img matListAvatar [src]="event.actor.avatarUrl" [alt]="event.actor.username" />
-        <h4 mat-line [matTooltip]="event.payload.title">{{ event.payload.title }}</h4>
-        <p mat-line>{{ event.createdOn | timeAgo }}</p>
-      </mat-list-item>
-    </mat-list>
-    <mat-list fxFlex dense>
-      <h3 mat-subheader>Pull Requests</h3>
-      <mat-list-item *ngFor="let pullRequest of repository.pullRequests | slice:0:5">
-        <img matListAvatar [src]="pullRequest.owner.avatarUrl" [alt]="pullRequest.owner.username" />
-        <h4 mat-line>{{ pullRequest.title }}</h4>
-        <p mat-line>{{ pullRequest.createdOn | timeAgo }}</p>
-      </mat-list-item>
-    </mat-list>
-    <mat-list fxFlex dense>
-      <h3 mat-subheader>Releases</h3>
-      <mat-list-item *ngFor="let release of repository.releases | slice:0:5">
-        <h4 mat-line>{{ release.title }}</h4>
-        <p mat-line>{{ release.createdOn | timeAgo }}</p>
-      </mat-list-item>
-    </mat-list>
+  <mat-card-content>
+    <div fxLayout="row" fxLayoutGap="2%" class="dashboard-cards-row">
+      <div fxFlex="24%">
```

Comment:
why `24%` and not `25%`?

Reply:
To maintain the ratio with above card. I have taken width 24% of below cards. If I change it to 25% then CSS will be distorted